src/main/java/org/opentripplanner/api/common/Crypto.java

```diff
@@ -0,0 +1,90 @@
+package org.opentripplanner.api.common;
+
+import org.apache.commons.codec.binary.Base64;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.crypto.Cipher;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Optional;
+
+public class Crypto {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Crypto.class);
+
+    private static SecretKeySpec keySpec;
+
+    private static String separator = "___-___";
+
+    static {
+
+        String secretKey = Optional.ofNullable(System.getenv("ENCRYPTION_SECRET_KEY"))
+                .filter(s -> !s.trim().isEmpty())
+                .orElseGet(() -> {
+                    LOG.error("No environment variable ENCRYPTION_SECRET_KEY defined! Falling back on default secret key!");
+                    return "very secret key";
```

Comment:
How about using a random key?

Reply:
That would make it harder to run the tests but I can try.

Reply:
Switched to random and set a fixed env variable for the test runs.